src/form.js

```diff
@@ -154,18 +154,18 @@ define([
     Form.prototype = {
         isValidHashtag: function(tag) {
             tag = this.normalizeHashtag(tag);
-            return this.hashtagDictionary.hasOwnProperty(tag) && _.isString(this.hashtagDictionary[tag]);
+            return this.hashtagDictionary.hasOwnProperty(tag) && !_.isFunction(this.hashtagDictionary[tag]);
```

Comment:
Having two types of values in `hashtagDictionary` seems like it could cause confusion, given the difference in meaning if the value is a string or function. I'm not sure I understand this enough to suggest another idea, but one that comes to mind is to put the different types of things in separate dictionaries so they don't need to be filtered like this. Would that make sense, or would it make things worse?